infra-aws/guides/create-asterion-infra-aws-org.org

```diff
@@ -0,0 +1,97 @@
+#+TITLE: Asterion infrastructure deployment
```

Comment:
Hey @shawngerrard - Nice work on the refactor to separate stack. I just noticed this new guide subdirectory with what appears to be a copy of the `README.org` from one level up?   

Can we please stick with just using the `README.org` at the higher level.